src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -95,15 +95,23 @@ public static void SetCurrentCircuitHost(CircuitHost circuitHost)
 
         public IDispatcher Dispatcher { get; }
 
-        public Task<IEnumerable<string>> PrerenderComponentAsync(Type componentType, ParameterCollection parameters)
+        public Task<ComponentRenderedText> PrerenderComponentAsync(Type componentType, ParameterCollection parameters)
         {
             return Dispatcher.InvokeAsync(async () =>
             {
                 var result = await Renderer.RenderComponentAsync(componentType, parameters);
+
+                await OnCircuitOpenedAsync(CancellationToken.None);
+
                 return result;
             });
         }
 
+        internal void Initialized()
+        {
+            _initialized = true;
+        }
```

Comment:
Where is this used?

Reply:
I merged it with the rest of the logic that got moved to CircuitHost from component hub. It was used in Connect